fbw-a32nx/src/systems/fmgc/src/flightplanning/new/AircraftConfigInterface.ts

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) 2021-2022 FlyByWire Simulations
+//
+// SPDX-License-Identifier: GPL-3.0
+
+export enum VnavDescentMode {
+  NORMAL,
+  CDA,
+  DPO,
+}
+
+export type AircraftConfig = {
```

Comment:
Use an interface. Also, rename the file to `AircraftConfig.ts` or `AircraftConfigTypes.ts`

Reply:
done, renamed to AircraftConfigTypes.ts